### PR TITLE
doc edits

### PR DESCRIPTION
**Pull request by aekiss**

### CODE REVIEW

**Comment by navidcy** [comment]
could you reopen this PR to `main` since it modifies the `README.md`?

**Review by navidcy — CHANGES_REQUESTED** [review]
@aekiss these are great additions, but since they are not on the `paper.md` file we should need to merge them on `#main` otherwise the changes will only appear once the JOSS paper is accepted. So if you could please, whenever you are done editing, open another PR from `joss-paper-ak2` branch to `main` that'd be great!

File: demos/premade_run_directories/readme_premade_rundirs.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
-
 ## Premade Run Directories
 
-These folders are used for the demo notebooks, and can be used as templates for setting up a new experiment. The difference in the choices is the surface forcing, affecting the `data_table`. 
+These folders are used for the demo notebooks, and can be used as templates for setting up a new experiment. The [documentation](https://github.com/COSIMA/regional-mom6/blob/main/docs/mom6-file-structure-primer.md) explains what all the files are for.
```
Comment (navidcy) [inline demos/premade_run_directories/readme_premade_rundirs.md:3]:
```suggestion
These folders are used for the demo notebooks, and can be used as templates for setting up a new experiment. The [documentation](https://regional-mom6.readthedocs.io/en/latest/mom6-file-structure-primer.html) explains what all the files are for.
```

File: demos/premade_run_directories/readme_premade_rundirs.md [hunk 2]
```diff
@@ -1,6 +1,5 @@
...
 
-The `common_files` folder contains all of the required files for a regional MOM6 run, including a `data_table` with constant surface forcing as a placeholder. If you call `.setup_rundir` and specify `jra` 
\ No newline at end of file
+The `common_files` folder contains all of the required files for a regional MOM6 run, including a `data_table` with constant surface forcing as a placeholder. The other two directories offer different choices for the surface forcing, affecting the `data_table`.
```
Comment (navidcy) [inline demos/premade_run_directories/readme_premade_rundirs.md:5]:
```suggestion
The `common_files` directory contains all of the required files for a regional MOM6 run, including a `data_table` with constant surface forcing as a placeholder. The other two directories offer different choices for the surface forcing, affecting the `data_table`.
```

**Comment by aekiss** [comment]
Oops, I just saw your posts @navidcy. I attempted a new PR here https://github.com/COSIMA/regional-mom6/pull/155 but it's including everything in `joss-paper` which isn't what we want.

Do I need to rebase or something?

**Comment by navidcy** [comment]
let me know when you are done editing in the `joss-paper-ak2` branch and I can sort it out

**Comment by navidcy** [comment]
I'm closing this. I opened #157 with all the changes you suggested here!

**Pull request closed without merging**